Hippo.Web/ClientApp/src/components/Account/RequestForm.tsx

```diff
@@ -40,10 +43,10 @@ export const RequestForm = () => {
     };
 
     fetchGroups();
-  }, [cluster]);
+  }, [clusterName]);
 
   const handleSubmit = async () => {
-    const req = authenticatedFetch(`/api/${cluster}/account/create`, {
+    const req = authenticatedFetch(`/api/${clusterName}/account/create`, {
```

Comment:
Can the cluster name contain invalid characters?
Maybe use a cluster slug like we do in some of our other team apps?

Reply:
It's only a variable name change, since `cluster` is now an object of type `Cluster`.